test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_test.clj

```diff
@@ -744,6 +748,26 @@
              (is (= 200 (:status @response)))
              (is (= hello-body (:body @response))))))))))
 
+(deftest double-stop-test
+  (testing "if the stop lifecycle is called more than once, we handle that gracefully and quietly"
+    (let [log-messages (atom [])]
+      (tk-log-testutils/with-logging-to-atom tk-log/root-logger-name log-messages
+        (let [app (tk-bootstrap/bootstrap-services-with-config
+                   [jetty9-service]
+                   {:webserver {:port 8080}})]
+          (tk-app/stop app)
+          (tk-app/stop app)))
+      ;; if we try to unregister the mbeans more than once Jetty will log a bunch
+      ;; of very loud and terrifying InstanceNotFoundExceptions
+      (let [log-event-filter #(and (= :warn (:level %))
+                                   (not (nil? (:exception %)))
+                                   (re-find #"InstanceNotFoundException"
+                                            (.getClassName (:exception %))))
```

Comment:
I'm guessing that there isn't anything better to test here, given what is stated in the commit message, but this seems really brittle - if anything happened to cause errors here again and if the error changed, we wouldn't have any notification (and, at least I think?, any other errors logged would be hidden in the test). This seems not great. Is there anything else we could test for? Even just that nothing is logged as a warning or an error? Or would that be too general and also brittle in another way?